.gitignore

```diff
@@ -4,3 +4,4 @@
 coverage/*
 notes.md
 tests/fixtures/cassette*
+composer.lock
```

Comment:
i think the .lock file adds more harm than doing benefit, but its your call really.

i updated this PR to not touch the ignore / .lock file